README.rst

```diff
@@ -313,7 +313,7 @@ Attributes:
 **templated_email_send_on_failure** (default: False):
     This attribute tells django-templated-email to send an email if the form is invalid.
 
-**templated_email_from_email** (default: **settings.TEMPLATED_EMAIL_FROM_EMAIL**):
+**templated_email_from_email** (default: **settings.TEMPLATED_EMAIL_FROM_EMAIL**. If it's also not defined, falls back to **DEFAULT_FROM_EMAIL**):
```

Comment:
I think we need to add a section with all the available settings and their descriptions. Check the ones mentioned in README, check the ones used in code, and add a new section to README above "Class Based Views" section.

Reply:
I agree. Then I think this overrides list can be removed to concentrate the params in one place.

Reply:
This specific list should be kept because it's the params of the CBV.

Reply:
Oops, I've misread your previous text. Ignore my suggestion.